arbitrum-docs/launch-orbit-chain/how-tos/customize-stf.mdx

```diff
@@ -170,8 +170,8 @@ docker build . --target nitro-node-dev --tag custom-nitro-node-dev
 docker run --rm --entrypoint cat custom-nitro-node-dev target/machines/latest/module-root.txt
 ```
 
-Once you have the WASM module root, you can put it on-chain by calling `setWasmModuleRoot(newWasmModuleRoot)` on the rollup contract as the owner.
-The rollup contract address can be found in the chain deployment info JSON.
+Once you have the WASM module root, you can put it on-chain by calling `setWasmModuleRoot(newWasmModuleRoot)` through the upgradeExecutor contract as the owner. To call this method, you need to set `target` as your rollup contract address. Ensure that `targetCallData` starts with `0x89384960` (this is the signature of `setWasmModuleRoot(byte32)`), and that it's followed by your WASM module root.
```

Comment:
Not very important, but can you mention that you need to use [executeCall](https://github.com/OffchainLabs/upgrade-executor/blob/main/src/UpgradeExecutor.sol#L73) in the UpgradeExecutor? ( That way the reader can easily understand the parameters you're mentioning )